perun-core/src/main/java/cz/metacentrum/perun/core/impl/UsersManagerImpl.java

```diff
@@ -916,6 +916,16 @@ public void checkUserExtSourceExists(PerunSession sess, UserExtSource userExtSou
 		if(!userExtSourceExists(sess, userExtSource)) throw new UserExtSourceNotExistsException("UserExtSource: " + userExtSource);
 	}
 
+	public boolean checkUserExtSourceExistsById(PerunSession sess, int id) throws InternalErrorException {
```

Comment:
Check methods do not return boolean. As we are using them, they always end with or without error without any returned output.